src/Tools/XmlDocumentationsUtility/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+using System.Xml;
+using System.IO;
+using NodeDocumentationUtility;
+
+namespace XmlDocumentationsUtility
+{
+    public class XmlDocumentationsUtility
+    {
+
+        static void Main(string[] args)
+        {
+            try
+            {
+                //remove and check for node documentation xml under Debug configuration
+                string xmlDir = Path.Combine(args[0], "bin", "AnyCPU", "Debug", "en-US");
```

Comment:
We shouldn't hardcode the path as Debug or Release. arg[0] should point to Dynamo bin location.